Tests/WebTest/Controller/RESTContentControllerTest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2015 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Bundle\ContentBundle\Tests\WebTest\Controller;
+
+use Symfony\Cmf\Component\Testing\Functional\BaseTestCase;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * @author Maximilian Berghoff <Maximilian.Berghoff@mayflower.de>
+ */
+class RESTContentControllerTest extends BaseTestCase
```

Comment:
Currently not sure if i should switch to behat tests as you did on resource-rest-bundle. So maybe that test will disappear.